.github/workflows/rust-audit.yml

```
@@ -0,0 +1,28 @@
+name: Rust security audit
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    paths:
+      - '**/Cargo.toml'
+      - 'Cargo.lock'
```

```suggestion
  push:
    branches: [ master ]
    paths:
      - '**/Cargo.toml'
  pull_request:
    paths:
      - '**/Cargo.toml'
```

Description of this PR says:

> Run audit test on PRs and push to master as well as nightly

but in this config, it will only be run on all branches of tokio-rs/tokio repo.

Also, tokio is a library and we do not commit cargo.lock. So, `paths: Cargo.lock` will not actually be triggered.